### ISSUE TITLE
xml:lang in comments in emmo-inferred

### ISSUE THREAD

**Issue by JoanaFMorgado** [emmo-repo/EMMO#88]
Dear @emanueleghedini , @jesper-friis ,

I hope you are doing well.
While using the emmo-inferred.owl file I notice some a minor inconsistency I guess.  For instance the second line of the following excerpt of code is ```xml:lang="unibo.it"``` instead of  ```xml:lang="en"```. Please let me know what you think.

``` 
Version 1.0.0-alpha2</owl:versionInfo>
        <rdfs:comment xml:lang="unibo.it">Contacts:
Gerhard Goldbeck
Goldbeck Consulting Ltd (UK)
email: gerhard@goldbeck-consulting.com
```

Many thanks!

Kind Regards,
Joana

**Comment by jesper-friis** [comment]
Dear @JoanaFMorgado 
It is an ond issue that has been fixed in upstream 1.0.0-alpha2. Since emmo-inferred.owl is a generated artefact, I would prefer to remove it from the repository and only use GitHub pages instead. You will find an resently updated version on 
https://emmo-repo.github.io/

Since we currently don't have a command-line tool for running FaCT++, we have to generate the inferred ontology manually. Before we get that and create a CI workflow for it, the inferred ontology may be slightly outdated.

@emanueleghedini, is it ok with you that I remove emmo-inferred.owl from the repo? It doesn't play nice with git. Since it is a few MB in size and will change a lot for each commit, it will pollute the history and make EMMO unnecessary heavy and slow to clone.

**Comment by jesper-friis** [comment]
I am closing this issue.
